source/integration_tools.html.md

```diff
@@ -0,0 +1,73 @@
+# Integration Tools
+
+Integration Tools is a section of Workshop (Welkin's codeless configuration tool), which allows you to configure Apps and other integrations. You can configure which systems have access to data within Welkin and how they access it.
+
+# Overview
+
+Integration Tools allows developers and program directors to manage their integrations with Welkin without contacting a member of the Welkin professional services team. This is a self serve tool for managing how data flows out of Welkin via Welkin's APIs.
+
+Integration Tools is broken into 3 main sections. Credentials for setting up access credentials. Webhooks for getting notifcations about changes to data in Welkin. Environments for selecting which Welkin instance/environment you want to configure integrations for.
```

Comment:
Integration Tools supports:
- Credentials for setting up access credentials. 
- Webhooks for getting notifcations about changes to data in Welkin. 
- Environments for selecting which Welkin instance/environment you want to configure integrations fo

Reply:
also notifications has a typo in your original text